src/screens/LockedWallet.js

```diff
@@ -77,6 +89,16 @@ class LockedWallet extends React.Component {
     $('#confirmResetModal').modal('show');
   }
 
+  /**
+   * When user clicks to change the wallet, it will change the current selected wallet
+   *
+   * @param {Object} e Event of when the link is clicked
+   */
+   changeWalletClicked = (e) => {
```

Comment:
I feel this method name is misleading. There is no click, it's when the select changes. The param docstring should be changed as well.

Reply:
Changed to `changeWalletHandler`